config/metadata_profiles/m3_profile.yaml

```diff
@@ -18,6 +18,8 @@ classes:
 contexts:
   flexible_context:
     display_label: Flexible Metadata Example
+  special_context:
```

Comment:
Corresponding changes to this profile should probably also be made in the .koppie (and possibly .dassie?) versions, as well as the spec version.